[2023-12-03 17:14:20,002] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_automation_daily.testing manual__2023-12-03T17:14:13.703056+00:00 [queued]>
[2023-12-03 17:14:20,042] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_automation_daily.testing manual__2023-12-03T17:14:13.703056+00:00 [queued]>
[2023-12-03 17:14:20,044] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-12-03 17:14:20,045] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-12-03 17:14:20,045] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-12-03 17:14:20,292] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): testing> on 2023-12-03 17:14:13.703056+00:00
[2023-12-03 17:14:20,391] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_automation_daily', 'testing', 'manual__2023-12-03T17:14:13.703056+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpan93bfqk', '--error-file', '/tmp/tmpomua1yoi']
[2023-12-03 17:14:20,394] {standard_task_runner.py:52} INFO - Started process 533 to run task
[2023-12-03 17:14:20,395] {standard_task_runner.py:80} INFO - Job 18: Subtask testing
[2023-12-03 17:14:21,526] {task_command.py:369} INFO - Running <TaskInstance: dag_automation_daily.testing manual__2023-12-03T17:14:13.703056+00:00 [running]> on host b69c54045741
[2023-12-03 17:14:22,539] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sebastiancahyoardhiiswara@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_automation_daily
AIRFLOW_CTX_TASK_ID=testing
AIRFLOW_CTX_EXECUTION_DATE=2023-12-03T17:14:13.703056+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-03T17:14:13.703056+00:00
[2023-12-03 17:14:22,542] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-03 17:14:22,543] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python codes/test.py']
[2023-12-03 17:14:22,612] {subprocess.py:85} INFO - Output:
[2023-12-03 17:14:28,908] {subprocess.py:92} INFO - mysql+mysqlconnector://root:mysql@mysql-db:3306/mysql
[2023-12-03 17:14:28,912] {subprocess.py:92} INFO - postgresql://***:***@postgres-db:5432/***
[2023-12-03 17:14:29,261] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-03 17:14:29,463] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_automation_daily, task_id=testing, execution_date=20231203T171413, start_date=20231203T171420, end_date=20231203T171429
[2023-12-03 17:14:29,553] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-03 17:14:29,681] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
